integrations/gitlab/gitlab_integration/ocean.py

```diff
@@ -27,6 +27,7 @@
 async def handle_webhook(group_id: str, request: Request) -> dict[str, Any]:
     event_id = f'{request.headers.get("X-Gitlab-Event")}:{group_id}'
     with logger.contextualize(event_id=event_id):
+        logger.info(f"Got an event from webhook. event_type and group_id: {event_id}")
```

Comment:
you are missing the webhook type

Reply:
I wonder if maybe this should be logged only when missed observer handler, wdyt?

Reply:
what do you mean by webhook type? event_id holds template of: <EVENT_TYPE>:<GROUP_ID>
webhook_id initialized with group_id. for example: Push Hook:834848

Reply:
for the second comment: are we sure there is no other case where an event will not print logs because it failed before reaching the handler? if the only thing we are sure that could happen is unsupported observer then I agree, but I believe this log should be there for any case whatsoever.